File: DatingApp-SPA/src/app/_models/photo.ts
/* Photo (TypeScript Interface)
 *
 * Used in order to specify a photo type.
 * These are used in the user type.
 *
 * Author:  JoNi
 * Date:    2019-12-13
 */
export interface Photo {
    id: number;
    url: string;
    description: string;
    dateAdded: Date;
    isMain: boolean;
}

File: DatingApp.API/Models/Photo.cs
using System;

namespace DatingApp.API.Models
{
    public class Photo
    {
        // User ID
        public int Id { get; set; }
        // The photo URL
        public string Url { get; set; }
        // Description
        public string Description { get; set; }
        // When the photo was added
        public DateTime DateAdded { get; set; }
        // If the photo is a main photo (displayed on member card etc)
        public bool IsMain { get; set; }
        // Used for cloudinary
        public string PublicId { get; set; }
        // Relation variables between the photo and the user
        public User User { get; set; }
        public int UserId { get; set; }
    }
}